WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -3905,7 +3904,6 @@ function WeakAuras.NewDisplayButton(data)
   frame.buttonsScroll:AddChild(displayButtons[id]);
   WeakAuras.AddOption(id, data);
   WeakAuras.SetIconNames(data);
-  WeakAuras.SortDisplayButtons();
```

Comment:
You need to look at all callers of NewDisplayButton to figure out whether any of them relied on those calls and test most of them.

E.g. OnDuplicateClick -> DuplicateAura -> NewDisplayButton seems to me like it should need a ScanForLoads for the new button + SortDisplayButtons (?)

NewAura might need those calls too.

Reply:
It seems that SortDisplayButtons isn't actually necessary in most cases. But ScanForLoads certainly is when creating an aura while options is open.

In any case, I think that I'll retract this change for now and do some more research. I'd like to get the other fix in sooner rather than later.

Reply:
Right, it looked easy enough, there aren't that many callers and looking through them most call both or at least one already anyway.

SortDisplayButtons is used to sort top level buttons, since those are sorted by name and to filter the buttons according to the search.

So, make sure to test that, e.g. for Duplicate button, with top-level auras "A 1", "A 2" exsisting, duplicating the first one will create a "A 3" which should end up at the end. With a filter for "1", and duplicate, the new aura should be invisible.